test/run.js

```diff
@@ -0,0 +1,14 @@
+const list = require('../list/list.json')
+const listSchema = require('../list/list.schema.json')
+const Ajv = require('ajv')
+
+const ajv = new Ajv()
+const validate = ajv.compile(listSchema)
+const isValid = validate(list)
+
+if (isValid) {
+  console.log('All good! :)')
+} else {
+  const errors = validate.errors
+  errors.forEach((error) => console.error(error))
```

Comment:
but dont we need a `1` exit code here? So the ci knows it failed?

Reply:
Yes. We already fixed it here: https://github.com/ahaeslich/ankelegt/commit/c20daadf902bafb0583504fc171a01829516c184 😉

Reply:
Ich bin so gut. G - U - H - T

Reply:
Einmal kurz ankeguckt und zack fehler gespottet

Reply:
> Einmal kurz ankeguck

Jetzt bist du erfolgreich "infiziert" 😄.

Reply:
Ihr habt mich ankestiftet